### ISSUE TITLE
AttributeError: 'DatetimeIndexResampler' object has no attribute 'progress_apply' - Use of swifter with resample()

### ISSUE THREAD

**Issue by AndyLuckert** [jmcarpenter2/swifter#115]
I'd appreciate that this error would be resolved or explained how it came about and what I can do myself to solve it as a user.

**My specifics as for the packages installed:**
```
$user: conda list

# Name                    Version                   Build  Channel
dask                      2.17.2                     py_0
pandas                    1.0.4                    pypi_0    pypi
python                    3.7.7           h191fe78_0_cpython
swifter                   0.304            py37hc8dfbb8_0    conda-forge
```

**My example which resulted in the above-mentioned error:**
```
df = df.swifter.resample("1h",
                         axis=0,
                         label="left",
                         closed="right",
                         loffset=None,
                         base=0).apply(np.max)

AttributeError: 'DatetimeIndexResampler' object has no attribute 'progress_apply'
```

**More details on the dataframe utilized:**
```
df.head()
                                                                           Hydrogen sulfide
Place                                    Time
Bioreactor_entrance/16-29-july-2018-pos1 2018-07-16 13:20:49.010000+02:00              0.19
                                         2018-07-16 13:21:49.019000+02:00              0.25
                                         2018-07-16 13:22:49.017000+02:00              0.31
                                         2018-07-16 13:23:49.014000+02:00              0.16
                                         2018-07-16 13:24:49.011000+02:00              0.19

df.info()
None
<class 'pandas.core.frame.DataFrame'>
MultiIndex: 71374 entries, ('Bioreactor_entrance/16-29-july-2018-pos1', Timestamp('2018-07-16 13:20:49.010000+0200', tz='Europe/Madrid')) to ('Bioreactor_entrance/3-apr-4-june-2019-pos4', Timestamp('2019-06-03 12:32:00+0200', tz='Europe/Madrid'))
Data columns (total 1 columns):
 #   Column            Non-Null Count  Dtype
---  ------            --------------  -----
 0   Hydrogen sulfide  71374 non-null  float64
dtypes: float64(1)
memory usage: 9.5+ MB

```

**Comment by jmcarpenter2** [comment]
Hi @AndyLuckert ,

Thanks so much for raising this issue and providing the code and data sample.

I was able to reproduce your error and found that the source of bug comes from the fact that I am using tqdm's `progress_apply` to provide a progress bar, but that functionality doesn't work for resampler objects. NOTE: the progress bar works when swifter chooses to use Dask to do the resample apply.

I will fix this bug by removing the progress bar for the pandas path of the apply ASAP.

Thanks so much for bringing this to my attention,
Jason

**Comment by jmcarpenter2** [comment]
@AndyLuckert,

Just released version `0.305` with the fix. To upgrade to a version that doesn't have this issue, please do either `pip install swifter==0.305` or `pip install -U swifter`.

Thanks again for letting me know!
Jason
